apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html

```diff
@@ -13,7 +13,8 @@
 
```

Comment:
Please find the rest of the comments before file `proforma-invoice.component.html` in PR:
[https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App/pull/5524/files](https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App/pull/5524/files)